keyboards/crkbd/keymaps/benrestech/rules.mk

```diff
@@ -0,0 +1,12 @@
+
+# Build Options
+#   change to "no" to disable the options, or define them in the Makefile in
+#   the appropriate keymap folder that will get included automatically
+#
+EXTRAKEY_ENABLE = yes     # Audio control and System control
+LTO_ENABLE      = yes     # Enable link time optimisation
+MOUSEKEY_ENABLE = yes     # Mouse keys
+NKRO_ENABLE     = yes     # Enable N-Key Rollover
+RGBLIGHT_ENABLE = yes     # Enable WS2812 RGB underlight.
+OLED_ENABLE     = yes     # Enable OLED
+OLED_DRIVER     = SSD1306 # Define driver for OLED
```

Comment:
```suggestion
RGBLIGHT_ENABLE = yes
OLED_ENABLE = yes
OLED_DRIVER = SSD1306
```

Reply:
I can see all the other removed options defined in the crkbd default makefile, what about the `LTO_ENABLE = yes     # Enable link time optimisation` option?

Reply:
LTO is already enabled for this board.

Reply:
Awesome, thanks!